databases/cockroachdb/README.md

```diff
@@ -0,0 +1,215 @@
+temp dir:            /Users/daniel/workspace/prisma-examples/databases/cockroachdb/cockroach-data/cockroach-temp960221969
+external I/O path:   /Users/daniel/workspace/prisma-examples/databases/cockroachdb/cockroach-data/extern
+store[0]:            path=/Users/daniel/workspace/prisma-examples/databases/cockroachdb/cockroach-data
+storage engine:      pebble
+status:              restarted pre-existing node
+clusterID:           dfa695a0-22e5-4356-8132-449169688432
+nodeID:              1
+```
+
+## 3. Create the database schema with the included SQL
+
+You will create the database schema using the [`dbinit.sql`](./dbinit.sql) file included in the repository.
+
+Run the following command from the `cockroachdb` folder:
+
+```
+cockroach sql --insecure -f dbinit.sql
+```
+
+The script will create the `prisma` database and create three tables.
+
+You should see the following output:
+
+```
+CREATE DATABASE
+Time: 116ms
+SET
+Time: 83ms
+CREATE TABLE
+Time: 183ms
+CREATE TABLE
+Time: 524ms
+CREATE TABLE
+Time: 569m
+```
+
+## 4. Configure the database connection URL
+
+Prisma will use the `DATABASE_URL` environment variable in `.env` in the `cockroachdb` folder to connect to the database.
+
+Create the file:
+
+```sh
+touch .env
+```
+
+Then add the following line:
+
+```
+DATABASE_URL=postgresql://root@localhost:26257/prisma?sslmode=disable
+```
+
+## 5. Introspect the database
+
+In this step, you will use the `prisma db pull` command to introspect your database and populate the Prisma schema.
+
+Run the following command:
+
+```sh
+npx prisma db pull
+```
+
+You should see the following output:
+
+```
+Environment variables loaded from .env
+Prisma schema loaded from prisma/schema.prisma
+Datasource "db": CockroachDB database "prisma", schema "public" at "localhost:26257"
+
+Introspecting based on datasource defined in prisma/schema.prisma …
+
+✔ Introspected 3 models and wrote them into prisma/schema.prisma in 334ms
+
+Run prisma generate to generate Prisma Client.
+```
+
+If you open the Prisma schema (`prisma/schema.prisma`) you should see 3 models.
+
+## 6. Rename relation fields in the Prisma schema
+
+Now that you have introspected your database, you will rename the relation fields in the Prisma schema so that its easier to access relations using the same field naming conventions.
+
+> **Note:** [Prisma-level relation fields](https://www.prisma.io/docs/concepts/components/prisma-client/working-with-prismaclient/use-custom-model-and-field-names#renaming-relation-fields) (sometimes referred to as "virtual relation fields") only exist in the Prisma schema, but do not actually manifest in the underlying database. You can therefore name these fields whatever you want.
+
+Open the Prisma schema and apply the following changes:
+
+```diff
+model Comment {
+  id        String   @id(map: "primary") @default(dbgenerated("gen_random_uuid()")) @db.Uuid
+  createdAt DateTime @default(now())
+  comment   String
```

Comment:
I think we can rename this to content instead.

Reply:
I prefer leaving it as `comment`, since that's what it is. 

Is there any particular reason to rename it to `content`?

Reply:
It seems a little odd that the model and field have the same name

Reply:
Sounds reasonable! I updated the column to `content` and the tests.